src/Util/TypeWriter.lua

```diff
@@ -0,0 +1,17 @@
+return function(label: TextLabel, text: string, delayBetweenChars: number?)
```

Comment:
This file is camelCase while the rest of the project is PascalCase. I don't think we should be inconsistent within the same, small project.